applications/FemToDemApplication/custom_processes/generate_initial_skin_DEM_process.cpp

```diff
@@ -149,4 +162,20 @@ int GenerateInitialSkinDEMProcess::GetMaximumDEMId()
     return max_id;
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+int GenerateInitialSkinDEMProcess::GetMaximumFEMId()
+{
+    int max_id = 0;
+    const auto it_FEM_node_begin = mrModelPart.NodesBegin();
+    // #pragma omp parallel for
+    for (int i = 0; i < static_cast<int>(mrModelPart.Nodes().size()); i++) {
```

Comment:
Still missing OMP max and min

Reply:
This could be a candidate to be implemented in  #943

Reply:
The mytical #943